core/src/main/java/io/aiven/klaw/model/response/EnvModelResponse.java

```diff
@@ -44,6 +44,10 @@ public class EnvModelResponse implements Serializable {
 
   private String topicsuffix;
 
+  private String topicregex;
+
+  private boolean topicAdvancedConfig;
```

Comment:
I can't find any dependencies for this

Reply:
This is in the Env Class and is required to let the Front end know if the topic Prefix/Suffix is being used (false) or if the topic Regex is being used (true) it is then used in the TopicRequestValidatorImpl to determine the correct validation to apply.